PopPUNK/__main__.py

```diff
@@ -587,13 +620,37 @@ def main():
             except:
                 sys.stderr.write("Isolates in subset not found in existing database\n")
             assert postpruning_combined_seq == viz_subset
+            
+            # Either use strain definitions, lineage assignments or external clustering
+            isolateClustering = {}
+            # Use external clustering if specified
+            if args.external_clustering:
+                cluster_file = args.external_clustering
+                isolateClustering = {'combined': readClusters(cluster_file, return_dict=True)}
+            elif args.viz_lineages:
+                cluster_file = args.viz_lineages
+                #isolateClustering = {'combined': readClusters(cluster_file, return_dict=True)}
+                isolateClustering = readLineages(cluster_file)
+            else:
+                # identify existing analysis files
+                model_prefix = args.ref_db
```

Comment:
Clearer as if/else

Reply:
Fixed in 8e0875ff87660230a3b41e195d93e3c2f49464a9.